armi/operators/tests/test_operators.py

```diff
@@ -103,37 +105,101 @@ def test_loadStateError(self):
         with self.assertRaises(RuntimeError):
             self.o.loadState(0, 1)
 
+    def test_setStateToDefault(self):
+
+        # reset the runType for testing
+        self.assertEqual(self.o.cs["runType"], "Standard")
+        self.o.cs = self.o.cs.modified(newSettings={"runType": "fake"})
+        self.assertEqual(self.o.cs["runType"], "fake")
+
+        # validate the method works
+        cs = self.o.setStateToDefault(self.o.cs)
+        self.assertEqual(cs["runType"], "Standard")
+
+    def test_snapshotRequest(self):
+        with TemporaryDirectoryChanger():
+            self.o.snapshotRequest(0, 1)
+
+
+class TestTightCoupling(unittest.TestCase):
+    def setUp(self):
+        self.cs = settings.Settings()
+        settings.setMasterCs(self.cs)
```

Comment:
@john-science this is the culprit.

Reply:
I mean, if this HAS to go in for this PR, I will hold my nose and do it.

But we WILL have to add it to the list to remove ASAP:

https://github.com/terrapower/armi/issues/930